ui-tests/src/test/resources/features/integrations/jira.feature

```diff
@@ -0,0 +1,217 @@
+#@sustainer: alice.rum@redhat.com
```

Comment:
add a space between `#` and `@` (don't know if it breaks something, just to be sure :grin:)

Reply:
add a link to the doc / steps needed to use the test